[TECH] Retirer la desactivation de l'espace surveillant (PIX-6615)

aceol opened the pull request:
## :christmas_tree: Problème
Il est possible de desactivé l'espace surveillant par centre de certif. On ne souhaite plus le permettre
## :gift: Proposition
Retirer la possibilité de desactiver

## :star2: Remarques
En attente: dois on pouvoir encore finaliser les sessions qui n'etait pas avec l'espace surveillant
**A supprimer:**
Le champ en BDD `certificationCenter.isSupervisorAccessEnabled` autorise l'espace surveillant pour toutes les sessions du centre de certification
**A ne pas supprimer?**
La table 'supervisor-access` enregistre les accès des surveillants à une session. 
La propriété 'session.hasSupervisorAccess` est alors envoyé au front **certif** pour déterminé si on doit gérer l'espace surveillant ou non. On peut ainsi garder une trace de l'activation de l'espace surveillant même si le champs `isSupervisorAccessEnabled` est mis à jour

## :santa: Pour tester
Finaliser une session sans espace surveillant et s'assurer que c'est ok (Dans les seeds la session 4 (sco) est à finaliser sans espace surveillant)
Creer une session, s'assurer qu'on a bien le code surveillant dans les infos de session
Passer la session avec l'espace surveillant
Finaliser la session

aceol committed:
:fire: api: Remove end test screen

aceol committed:
:fire: api: Remove end-test-screen from attendance sheet

aceol committed:
:recycle: api: Update tests with end test screen enabled

aceol committed:
:fire: api: Remove end test screen removal from tests

aceol committed:
:fire: api: Remove end test screen removal from infra

aceol committed:
:fire: api: Remove end test screen removal from app and scripts

aceol committed:
:fire: api: Remove end-test-screen from domain

pix-service commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr5500.review.pix.fr)
  * [App (.org)](https://app-pr5500.review.pix.org)
  * [Orga](https://orga-pr5500.review.pix.fr)
  * [Certif](https://certif-pr5500.review.pix.fr)
  * [Admin](https://admin-pr5500.review.pix.fr)
  * [API](https://api-pr5500.review.pix.fr/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr5500/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr5500/environment)

aceol committed:
:fire: admin: Remove toggle on certification center supervisor access

aceol committed:
:card_file_box: api: Update seeds with last assessment results

The script does not have to be run apart from the seeds hence can be simplified

aceol committed:
:fire: mon-pix: Drop end test screen feature

aceol committed:
:recycle: certif: Update test description

aceol committed:
:fire: certif: Drop end test screen feature

aceol commented:
Il faudrait extraire la migration (suppression de la colonne) pour la faire sur la MEP suivante

alexandrecoin commented on certif/tests/integration/components/session-finalization/completed-reports-information-step_test.js:
```diff
@@ -140,7 +140,7 @@ module('Integration | Component | SessionFinalization::CompletedReportsInformati
   });
 
-  module('when the end test screen removal feature is disabled', function () {
+  module('when the end test screen removal feature was disabled during the certification', function () {
```
Question: Pourquoi garder un intitulé de test qui mentionne un FT supprimé ?

aceol replied:
J'ai changé le nom du test justement parce que l'intention est de tester à la finalisation si la feature était desactivée lors du passage de la certif (avant T)
On recupere cette information non pas via le flag sur la session (qui peut changer de valeur entre le passage de la certif et lors de la finalisation) mais via la table d'acces des surveillants à la session

alexandrecoin reviewed (approved):
lgtm

P-Jeremy commented:
Test ok pour session 4 sco

![image](https://user-images.githubusercontent.com/37305474/214081923-3976fa05-20b3-4f0c-947e-f0b70a514e91.png)

aceol committed:
:sparkles: api: Sort query to avoid flaky tests

aceol committed:
:recycle: api: Empty tables after tests

https://github.com/1024pix/pix/pull/5300
